CITATION.cff

```diff
@@ -0,0 +1,8 @@
+@software{RandLAPACK2023,
+  author = {Maksim, Melnichenko and Riley, Murray and Burlen, Loring and Parth, Nobel},
```

Comment:
@TeachRaccooon only put comments between first and last names if the first name appears _after_ the last name. So "Murray, Riley" is correct but "Riley, Murray" is not.

Reply:
Fixed.